assignments/objects.js

```diff
@@ -49,16 +96,42 @@ let example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-let parent = {}
+let parent = {
```

Comment:
💯 Totally enjoyed reviewing your understanding of objects. You have a very good grasp on how to utilize them. This will come in handy in the future.